engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41710to41800.java

```diff
@@ -98,6 +104,37 @@ public void updateSystemVmTemplates(Connection conn) {
         }
     }
 
+    protected void decryptGlobalConfigurationValues(Connection conn) {
```

Comment:
the method name is misunderstanding.
only the Hidden/Secure configuration values in account and domain details are NOT decrypted, all others are decrypted.